packages/hardhat/contracts/HatchTemplate.sol

```diff
@@ -139,14 +141,18 @@ contract HatchTemplate is BaseTemplate, AppIdsXDai {
     * @param _redeemableTokens Array of initially redeemable tokens
     * @param _tollgateFeeToken The token used to pay the tollgate fee
     * @param _tollgateFeeAmount The tollgate fee amount
+    * @param _scoreToken Token for Hatch Oracle used to determine contributor allowance
+    * @param _hatchOracleRatio Hatch Oracle ratio used to determine contributor allowance
+    * @param _voteOpenAfterPeriod Time period in which vote creation will not be allowed
```

Comment:
I think this would be the opposite: The time period where the vote creation will BE allowed